building/devices/versal2.rst

```diff
@@ -0,0 +1,42 @@
+.. _versal2:
+
+################
+AMD Versal Gen 2
+################
+The following instructions demonstrate how to obtain the OP-TEE source code and build it for the AMD Versal Gen 2 platform.
+
+Supported boards
+****************
+AMD Versal Gen 2 adaptive SoCs combine world-class programmable logic from AMD with a new high-performance processing system of integrated Arm CPU.
```

Comment:
Not very useful in this (technical) documentation. Instead, please mention (or link to) which board precisely are supported. Readers should be able to know which board model to purchase if they want to run OP-TEE.